README.md

```
@@ -1,21 +1,71 @@
+1. Navegar a la carpeta de la app:
+    ```bash
+    cd testing-app
+    ```
+
+2. Instalar las dependencias:
+    ```bash
+    npm install
+    ```
+
+3. Crear un archivo `.env` en la raíz de la `testing-app` con el siguiente contenido:
+    ```env
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)